packages/ag-charts-community/src/util/json.ts

```diff
@@ -111,26 +117,30 @@ export function shallowClone<T>(source: T): T {
  *
  * @param json to traverse
  * @param visit callback for each non-primitive and non-array object found
- * @param opts
- * @param opts.skip property names to skip when walking
- * @param jsons to traverse in parallel
+ * @param skip property names to skip when walking
+ * @param parallelJson to traverse in parallel
  */
-export function jsonWalk<T>(json: T, visit: (...nodes: T[]) => void, opts?: { skip?: string[] }, ...jsons: T[]) {
+export function jsonWalk<T, C>(
```

Comment:
More micro-optimisations - mostly removing use of `...` and `Array.includes()` in preference of a single param and `Set.has()` respectively (also removal of lambda use)